assets/js/components/wp-dashboard/WPDashboardWidgets.js

```diff
@@ -84,9 +125,14 @@ const WPDashboardWidgets = () => {
 					) }
 				</div>
 			) }
-			{ analyticsModuleActive && analyticsModuleConnected && (
-				<WPDashboardPopularPagesWidget />
-			) }
+
+			{ analyticsModuleActive && analyticsModuleConnected && ! shouldCombineAnalyticsArea2 && <WPDashboardPopularPagesWidget /> }
+
+			<div className={ HIDDEN_CLASS }>
+				{ shouldCombineSearchConsoleWidgets && <WPDashboardClicksWidget /> }
+				{ shouldCombineAnalyticsArea1 && <WPDashboardSessionDurationWidget /> }
+				{ shouldCombineAnalyticsArea2 && <WPDashboardPopularPagesWidget /> }
```

Comment:
I think in this line we still also need the conditions that Analytics is active and connected that the widget has above - otherwise this may render without Analytics being active/connected, e.g. causing API requests to be made that could result in errors.